src/main/java/org/alfresco/repo/security/authority/AuthorityDAOImpl.java

```diff
@@ -1656,6 +1657,28 @@ public void onUpdateProperties(NodeRef nodeRef, Map<QName, Serializable> before,
         }
     }
 
+    public void onCreateNode(ChildAssociationRef childAssocRef)
+    {
+        // Restrict creation of group name that contain invalid characters.
+        NodeRef groupRef = childAssocRef.getChildRef();

```

Comment:
Please change the variables to `authorityRef` and `authorityName`

Reply:
Done.